src/spec/usecase/usecase_a.js

```diff
@@ -0,0 +1,97 @@
+export default (Passable) => {
+
+    const response = Passable('case_a', (pass, enforce) => {
+        pass('field_1', 'should be a string of 5 chars', () => {
+            enforce('hello').allOf({
+                sizeEquals: 5,
+                isString: true
+            });
+        });
+
+        pass('field_2', 'must be a number smaller than 90', () => {
+            enforce(99).allOf({
+                smallerThan: 90,
+                isNumber: true
+            });
+        });
+
+        pass('field_3', 'must be either "a" or "b"', () => {
+            enforce('a').allOf({
+                inside: ['a', 'b']
+            });
+        });
+
+        pass('field_4', 'should be either "a" or "b"', 'warn', () => {
+            enforce('c').allOf({
+                inside: ['a', 'b']
+            });
+        });
+
+        pass('field_5', 'Must either be a number or a string. Always smaller than 5', () => {
+            enforce('log').anyOf({
+                isNumber: true,
+                isString: true
+            }).allOf({
+                smallerThan: 5
```

Comment:
How is this passing? Is Passable measuring String length if this test is tested against a String and not a Number?
Same for the other cases.

Reply:
Yes.
> Returns true if a given value is smaller than another value. The values do not have to be of the same type
https://fiverr.github.io/passable/enforce/rules/size/smaller_than/index.html

More specifically, it internally uses the `getSize` helper function which determines the size/length of an item.

https://github.com/fiverr/passable/blob/master/src/enforce/rules/helpers/get_size/index.js